src/main/java/net/revelc/code/formatter/FormatterMojo.java

```diff
@@ -792,7 +792,7 @@ protected void doFormatFile(final File file, final ResultCollector rc, final Pro
 
         // Process the result type
         if (Result.SKIPPED.equals(result)) {
-            rc.unchangedCount++;
+            rc.skippedCount++;
```

Comment:
I wasn't sure if this line should be changed or not. I changed it for testing and forgot to revert it before committing. If this change should be removed, please let me know.

Reply:
Change it back for purpose of this pull request.  I do want to look more at this part after as it's still confusing on naming.

Reply:
Updated. Sorry about that, I should have just done it last night :)